ziti/edge/api/client.go

```diff
@@ -167,11 +169,39 @@ func (c *ctrlClient) CreateSession(svcId string, kind edge.SessionType) (*edge.S
 	return decodeSession(resp)
 }
 
```

Comment:
Do we need to fallback to individual submissions if the endpoint isn't supported? Could remember if you get a 404 and call the individual submission api instead.

Reply:
Sure should and now does.